exercises/error-handling/src/test/java/ErrorHandlingTest.java

```diff
@@ -0,0 +1,66 @@
+import junit.framework.AssertionFailedError;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Optional;
+
+public class ErrorHandlingTest {
+
+    private ErrorHandling errorHandling;
+
+    @Before
+    public void setUp() {
+        errorHandling = new ErrorHandling();
+    }
+
+    @Test(expected = Exception.class)
```

Comment:
We've decided to use [`ExpectedException`](https://junit.org/junit4/javadoc/4.12/org/junit/rules/ExpectedException.html) on this track instead of `@Test(expected = ...)`. This is because you can do more with `ExpectedException` such as assert what the error message is (which would be useful in your next test) 🙂 See [the phone number tests](https://github.com/exercism/java/blob/3631108c0d2fd245f603017fb86abdf462c7d10a/exercises/phone-number/src/test/java/PhoneNumberTest.java) for an example of how it's used 🙂

Reply:
OK. I didn't notice that guideline 🙂 I will fix it.